.jenkins

```diff
@@ -137,7 +137,7 @@ pipeline {
                               cmake \
                                 -DCMAKE_BUILD_TYPE=Debug \
                                 -DCMAKE_CXX_COMPILER=clang++ \
-                                -DCMAKE_CXX_FLAGS="-Wno-unknown-cuda-version -Werror" \
+                                -DCMAKE_CXX_FLAGS="-Wno-unknown-cuda-version -Werror -Wno-undefined-internal -Wno-pass-failed" \
```

Comment:
no-undefined-internal : avoid warnings on internal linkage due to the is_omp_initial_device2 that is inside the declare variant directives 

no-pass-failed : avoids warnings arising from simd directives in debug mode.

Reply:
Please amend these comments to 4e2ad4fa25af4699cb313f5af1f97f5ca0c9b790 commit message